ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureActivity.java

```diff
@@ -40,15 +40,23 @@ void onCreateInternal(CardinalClient cardinalClient) {
 
         ThreeDSecureResult threeDSecureResult = extras.getParcelable(EXTRA_THREE_D_SECURE_RESULT);
         if (threeDSecureResult != null) {
-            cardinalClient.continueLookup(this, threeDSecureResult, this);
+            try {
+                cardinalClient.continueLookup(this, threeDSecureResult, this);
+            } catch (BraintreeException e) {
```

Comment:
Which errors are thrown by Cardinal.getInstance().cca_continue function? Can it throw RuntimeException?